server/apiserver/apiserver.go

```diff
@@ -74,3 +80,74 @@ func (api *ApiServer) Headers(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 }
+
+// Upgrader "upgrades" HTTP endpoint to WS endpoint, this requires a Read and Write buffer size
+var upgrader = websocket.Upgrader{} // use default options
+
+// define a reader which will listen for new messages being sent to our WebSocket endpoint
+func reader(conn *websocket.Conn) {
+	for {
+		// read in a message
+		// messageType is an int with value websocket.BinaryMessage or websocket.TextMessage
+		// p is []byte
+		messageType, p, err := conn.ReadMessage()
```

Comment:
```suggestion
		_, p, err := conn.ReadMessage()
```
If we are getting unused variable errors and need to print it out to get rid of it, we can use go's dummy assignment to not make a variable for it